### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by ondrahlavac**

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -263,8 +263,12 @@ function exitHandler(options, err) {
   // pro všechny kanály kde jsme aktuálně přítomní
   // bot.say(messages[0]);
 
-  if (options.cleanup) console.log('clean');
-  if (err) console.log(err.stack);
+  if (options.cleanup) {
+    console.log('clean');
```
Comment (babant) [inline bot.js]: Codacy Issue found: [Unexpected console statement.](https://www.codacy.com/app/ondra/obed-o-bot/file/2229899613/issues/source?bid=3165659&fileBranchId=3175493#l267)

File: bot.js [hunk 2]
```diff
@@ -263,8 +263,12 @@ function exitHandler(options, err) {
...
+  }
+  if (err) {
+    console.log(err.stack);
```
Comment (babant) [inline bot.js]: Codacy Issue found: [Unexpected console statement.](https://www.codacy.com/app/ondra/obed-o-bot/file/2229899613/issues/source?bid=3165659&fileBranchId=3175493#l270)

File: bot.js (hunk 1 above)
Comment (babant) [inline bot.js]: Codacy Issue found: [Unexpected console statement.](https://www.codacy.com/app/ondra/obed-o-bot/file/2229899827/issues/source?bid=3165659&fileBranchId=3175493#l267)

File: bot.js (hunk 2 above)
Comment (babant) [inline bot.js]: Codacy Issue found: [Unexpected console statement.](https://www.codacy.com/app/ondra/obed-o-bot/file/2229899827/issues/source?bid=3165659&fileBranchId=3175493#l270)

File: bot.js [hunk 3]
```diff
@@ -263,11 +263,15 @@ function exitHandler(options, err) {
...
+  if (options.cleanup) {
+    //console.log('clean');
+  }
+  if (err) {
+    //console.log(err.stack);
+  }
   // pokud ukončujeme bota pomocí Ctrl-C, oznámíme to na Slacku a odpojíme se.
   if (options.exit) {
-    console.log(messages[0]);
+    console.log(messages[0]); // TODO: Nahradit console.log nějakým logovacím middlewarem.
```
Comment (babant) [inline bot.js]: Codacy Issue found: [Unexpected console statement.](https://www.codacy.com/app/ondra/obed-o-bot/file/2229907379/issues/source?bid=3165659&fileBranchId=3175493#l274)

File: bot.js [hunk 4]
```diff
@@ -258,16 +258,20 @@ controller.hears(
...
```
Comment (ondrahlavac) [inline bot.js:274]: Codacy Issue found: [Unexpected console statement.](https://www.codacy.com/app/ondra/obed-o-bot/file/2229908788/issues/source?bid=3165659&fileBranchId=3175493#l274)

**Pull request merged**